nucliadb_models/nucliadb_models/activity.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (C) 2021 Bosutech XXI S.L.
+#
+# nucliadb is offered under the AGPL v3.0 and as commercial software.
+# For commercial licensing, contact us at info@nuclia.com.
+#
+# AGPL:
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+from enum import Enum
+
+from pydantic import BaseModel, Field
+
+
+class ResourceOperationType(str, Enum):
+    CREATED = "created"
+    MODIFIED = "modified"
+    DELETED = "deleted"
+
+
+class NotificationAction(str, Enum):
+    COMMIT = "commit"
+    INDEXED = "indexed"
+    ABORTED = "aborted"
+
+
+class ResourceNotification(BaseModel):
```

Comment:
TODO: re-model this to have a more generic notification stream (not only for resource activity, but potentially for other things)

TODO: Confirm with @ebrehault that the model and the UX is OK